src/components/ActivityFeed/ActivityFeed.js

```diff
@@ -392,20 +431,24 @@ class ActivityFeed extends React.Component<Props, State> {
       return true;
     });
 
-    const processedHistory = additionalFiltering ? additionalFiltering(filteredHistory) : filteredHistory;
+    this.feedData = additionalFiltering ? additionalFiltering(filteredHistory) : filteredHistory;
```

Comment:
Mutating the component status on the `render` feels weird. Not sure what would be a good solution, but, isn't there a chance we can maybe move this whole part to its parent and pass the result as `props`? Probably the parent can filter the items itself, which would allow it to calculate the height, instead of having the flow go back and forth.

Reply:
Yeah, we could. Although, this component is used across multiple screens while holding almost the same information so it's connected to this component and are handled inside of it.